### PR TITLE
Add govuk-lint

### PR DESCRIPTION
**Pull request by gpeng**
Setup Jenkins to run govuk-lint to avoid lint-esque commenting on PRs, improve code consistency and respect the gov.uk style guide.

All the juice is in the first commit. The last two are noisy 'fix all the things' commits.

### CODE REVIEW

File: app/models/document.rb [hunk 1]
```diff
@@ -106,14 +106,12 @@ def humanized_attributes
...
-      {
-        content_id: payload.content_id,
-        title: payload.title,
-        summary: payload.description,
-        body: payload.details.body,
-        publication_state: payload.publication_state,
-        public_updated_at: payload.public_updated_at
-      }
+      content_id: payload.content_id,
+title: payload.title,
```
Comment (danielroseman) [inline app/models/document.rb]: this indentation isn't right.

File: app/models/manual.rb [hunk 2]
```diff
@@ -54,13 +50,9 @@ def sections
...
 
-  def organisation_content_ids
-    @organisation_content_ids
-  end
+  attr_reader :organisation_content_ids
 
-  def organisation_content_ids=(organisation_content_ids)
-    @organisation_content_ids = organisation_content_ids
-  end
+  attr_writer :organisation_content_ids
```
Comment (danielroseman) [inline app/models/manual.rb]: use attr_accessor instead?

File: app/models/manual.rb [hunk 3]
```diff
@@ -136,14 +128,12 @@ class RecordNotFound < StandardError; end
...
-      {
-        content_id: payload["content_id"],
-        title: payload["title"],
-        summary: payload["description"],
-        body: payload["details"]["body"],
-        publication_state: payload["publication_state"],
-        public_updated_at: payload["public_updated_at"],
-      }
+      content_id: payload["content_id"],
+title: payload["title"],
```
Comment (danielroseman) [inline app/models/manual.rb]: again very odd indentation

File: app/models/section.rb [hunk 4]
```diff
@@ -41,25 +41,17 @@ def public_updated_at=(timestamp)
     @public_updated_at = Time.parse(timestamp.to_s) unless timestamp.nil?
   end
 
-  def manual_content_id
-    @manual_content_id
-  end
+  attr_reader :manual_content_id
```
Comment (danielroseman) [inline app/models/section.rb]: attr_accessor here and below

File: app/models/section.rb [hunk 5]
```diff
@@ -91,15 +83,13 @@ def self.from_publishing_api(content_id:)
...
-        content_id: payload["content_id"],
-        title: payload["title"],
-        summary: payload["description"],
-        body: payload["details"]["body"],
-        publication_state: payload["publication_state"],
-        public_updated_at: payload["public_updated_at"],
-        manual_content_id: payload["links"]["manual"].first
-      }
+      content_id: payload["content_id"],
+title: payload["title"],
```
Comment (danielroseman) [inline app/models/section.rb]: indentation

File: spec/models/attachment_spec.rb [hunk 6]
```diff
@@ -1,20 +1,15 @@
...
+      Attachment.new(file: ActionDispatch::Http::UploadedFile.new(tempfile: "spec/support/images/cma_case_image.jpg", filename: File.basename("spec/support/images/cma_case_image.jpg"), type: "image/jpeg"),
+       title: 'test attachment')
     }
 
     it "should set content_id and file attributes with data from attachments new form" do
-      attachment = Attachment.new({
-        file: ActionDispatch::Http::UploadedFile.new(tempfile: "spec/support/images/cma_case_image.jpg", filename: File.basename("spec/support/images/cma_case_image.jpg"),type: "image/jpeg"),
-        title: 'test attachment'
-      })
+      attachment = Attachment.new(file: ActionDispatch::Http::UploadedFile.new(tempfile: "spec/support/images/cma_case_image.jpg", filename: File.basename("spec/support/images/cma_case_image.jpg"), type: "image/jpeg"),
```
Comment (danielroseman) [inline spec/models/attachment_spec.rb]: this is less readable than before

File: spec/models/attachment_spec.rb [hunk 7]
```diff
@@ -27,14 +22,12 @@
 
     it "should set content_id, title, url, created_at and updated_at with data from publishing-api" do
       content_id = SecureRandom.uuid
-      attachment = Attachment.new({
-        title: 'test attachment',
+      attachment = Attachment.new(title: 'test attachment',
```
Comment (danielroseman) [inline spec/models/attachment_spec.rb]: more strange indentation

**Comment by gpeng** [comment]
Thanks @danielroseman. I've addressed your comments in f851641

**Pull request merged**